JeMPI_Apps/JeMPI_Bootstrapper/src/main/resources/data/postgres/audit-schema.sql

```diff
@@ -2,10 +2,9 @@ CREATE TABLE IF NOT EXISTS audit_trail (
     id             UUID         NOT NULL DEFAULT gen_random_uuid(),
     insertedAt     TIMESTAMP    NOT NULL DEFAULT now(),
     createdAt      TIMESTAMP    NOT NULL,
-    interactionID  VARCHAR(64),
-    goldenID       VARCHAR(64),
-    event          VARCHAR(256),
+    eventType      VARCHAR(256),
+    eventData      TEXT,
```

Comment:
Could we JSONB here, and filter as appropriate below. In `JeMPI_Apps/JeMPI_LibAPI/src/main/java/org/jembi/jempi/libapi/PsqlAuditTrail.java`